modules/core-updates/9.0.md

```diff
@@ -117,6 +117,95 @@ Display event listeners for the back office in `dev` environment (default value
 - [Experimental front container](https://github.com/PrestaShop/PrestaShop/pull/32719)
 - [Admin API has dedicated kernel, we now have three distinct configurations](https://github.com/PrestaShop/PrestaShop/pull/35515)
 
+### Symfony layout
+
+All the back office pages share a common layout, composed of a few elements:
+- the `<head>` element that includes all the CSS and JS (among other things)
+- the side navigation menu
+- the header of the page which is itself composed of
+- quick accesses
+- search form
+- notifications center
+- employee dropdown
+- the multistore header (when multistore is enabled)
+- the page toolbar (which includes breadcrumb and top action buttons)
+- the footer (which only contains a displayBackOfficeFooter display hook)
+
+Until PrestaShop 8.1 all these common elements were handled by legacy code, so even on migrated page there was always a background legacy controller based on `AdminController` in charge of building the layout data and rendering it. Once the layout was rendered the central content of Symfony pages was included in the middle of it. It means no page was completely free from the legacy controllers which would ultimately block the end of migration where they are meant to disappear completely.
+
+In PrestaShop 9.0 all this layout part is now fully handled by Symfony, we use <a href="https://symfony.com/bundles/ux-twig-component/current/index.html" target="_blank">Twig components</a> to render each element independently. The code is, therefore, easier to understand, a component class is responsible for fetching/building the data while the actual rendering is based on Twig. See our <a href="https://github.com/PrestaShop/PrestaShop/tree/develop/src/PrestaShopBundle/Twig/Component" target="_blank">Layout components</a> for more details.
+
+In legacy pages the same principle is used, Symfony is now in charge of rendering all the layout, and we use some <a href="https://github.com/PrestaShop/PrestaShop/tree/develop/src/PrestaShopBundle/Twig/Component/Legacy" target="_blank">Legacy layout components</a> that follow the same architecture but render different twig templates to fit with the old **default theme** (that is based on old bootstrap and includes old legacy helpers like form and lists).
+
+**So what changes for my module pages?**
+
+We refactored this layout with maximum backward compatibility in mind, the HTML layout itself had minimum changes (in both migrated and legacy pages), we tried to keep the same hooks in both the PHP code and the twig templates. We even introduced a fake legacy controller in migrated pages that has no logic but is kept mostly as a DTO to contain things like CSS files and JS files, because that's where most modules add their content.
```

Comment:
```suggestion
We refactored this layout with maximum backward compatibility in mind. The HTML layout itself had minimum changes (in both migrated and legacy pages). We tried to keep the same hooks in both the PHP code and the Twig templates. We even introduced a fake legacy controller in migrated pages that has no logic but is kept mostly as a [DTO](https://en.wikipedia.org/wiki/Data_transfer_object) to contain things like CSS files and JS files because that's where most modules add their content.
```